torch/_inductor/codegen/cuda/cutlass_utils.py

```diff
@@ -50,33 +50,25 @@ def try_import_cutlass() -> bool:
     # TODO(ipiszy): remove this hack when CUTLASS solves Python scripts packaging structure issues.
 
     cutlass_py_full_path = os.path.join(
-        inductor_cuda_config.cutlass_dir, "tools/library/scripts"
+        inductor_cuda_config.cutlass_dir, "python/cutlass_library"
     )
     tmp_cutlass_py_full_path = os.path.abspath(
-        os.path.join(cache_dir(), "torch_cutlass_script")
+        os.path.join(cache_dir(), "torch_cutlass_library")
     )
 
     if os.path.isdir(cutlass_py_full_path):
-        cutlass_file_names = [
-            file_name
-            for file_name in os.listdir(cutlass_py_full_path)
-            if file_name.endswith(".py")
-        ]
-        cutlass_module_names = [file_name[:-3] for file_name in cutlass_file_names]
-        if not os.path.isdir(tmp_cutlass_py_full_path):
-            os.mkdir(tmp_cutlass_py_full_path)
-        for file_name in cutlass_file_names:
-            _gen_cutlass_file(
-                file_name,
-                cutlass_module_names,
-                cutlass_py_full_path,
-                tmp_cutlass_py_full_path,
+        if tmp_cutlass_py_full_path not in sys.path:
```

Comment:
Would be nice if that would be possible, but sadly there are lots of things in there that we don't want to place in sys.path ( like a "docs" directory or a "setup.py" ), see contents of https://github.com/NVIDIA/cutlass/tree/main/python 

once cutlass_library is pip installeable and part of requirements.txt, we will have a cleaner solution. This one at least does not require to actually copy and/or modify source files.

Reply:
Update: I had to revert this back to the old behavior, since Cutlass 3.2.1 is leading to issues with flash attention kernels that  are not resolved yet.